### PR TITLE
MEN-3880 go vet todos and go fmt

### PR DESCRIPTION
**Pull request by merlin-northern**
ChangeLog:none
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

**Review by alfrunes — APPROVED** [review]

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -47,7 +47,8 @@ test:unit:
     # also including their coverage may introduce to much noice. Concentrate on the coverage of local packages.
     # Execute go test on every local subpackage (resolved as dependencies) and generate covreage report for each.
     # Test packages pararell (xargs -P)
-    - go list ./... | grep -v vendor | xargs -n1 -I {} -P 4 go test -v -covermode=atomic -coverprofile=../../../{}/coverage.txt {} || exit $?
+    - sed -i -e 's/CipherString = DEFAULT@SECLEVEL=2/# CipherString = DEFAULT@SECLEVEL=2/' /etc/ssl/openssl.cnf
+    - go list ./... | grep -v vendor | xargs -n1 -I {} -P 4 go test -parallel 1 -count 1 -v -covermode=atomic -coverprofile=../../../{}/coverage.txt {} || exit $?
```
Comment (oleorhagen) [inline .gitlab-ci.yml]: Why this instead of regular `go test` on all the packages? it runs packages separately by default I think

Comment (merlin-northern) [reply .gitlab-ci.yml]: to be completely honest, I copied that from go-lib-micro do you have a suggestion how to change it?

Comment (oleorhagen) [reply .gitlab-ci.yml]: I think you might just as well do something along the lines of:
```
go test -coverpkg ./... ./...
```

Comment (merlin-northern) [reply .gitlab-ci.yml]: sure. not a problem. fixed.

**Review by oleorhagen — APPROVED** [review]

**Pull request merged**